x/tokenfactory/keeper/keeper_test.go

```diff
@@ -7,13 +7,16 @@ import (
 	"github.com/stretchr/testify/suite"
 
 	"github.com/osmosis-labs/osmosis/v7/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/keeper"
 	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types"
 )
 
 type KeeperTestSuite struct {
 	apptesting.KeeperTestHelper
 
-	queryClient types.QueryClient
+	queryClient  types.QueryClient
+	msgServer    types.MsgServer
```

Comment:
oh interesting, we should definitely add this to our default test suite code structure everywhere!